test/msd_test.py

```diff
@@ -398,7 +399,7 @@ def test_mass_storage(workspace, parent_test):
 
             # Sanity check the regions are contiguous
             if region_index:
```

Comment:
Minor note: Use a "> 0" comparison since, semantically, `region_index` is not a Boolean value.